src/Components/Users/UserProfile.tsx

```diff
@@ -164,8 +164,8 @@ export default function UserProfile() {
 
           if (altPhoneNumber) {
             alt_is_valid = altPhoneNumber.isValid();
-            if (alt_is_valid)
-              alt_is_valid = altPhoneNumber.getType() === "MOBILE";
+            if (alt_is_valid) console.log("Type is ", altPhoneNumber.getType());
+            alt_is_valid = altPhoneNumber.getType() === "MOBILE";
```

Comment:
```suggestion
            if (alt_is_valid)
              alt_is_valid = altPhoneNumber.getType() === "MOBILE";
```

Don't leave `console.log`s

Reply:
Removed `console.log`